lib/node_modules/@stdlib/stats/base/dists/uniform/cdf/benchmark/benchmark.js

```diff
@@ -36,11 +36,12 @@ bench( pkg, function benchmark( b ) {
 	var y;
 	var i;
 
+	x = uniform( -10.0, 10.0 );
+	min = uniform( -20.0, 0.0 );
+	max = uniform( min, min + 40.0 );
+
```

Comment:
```suggestion
	len = 100;
	x = new Float64Array( len );
	min = new Float64Array( len );
	max = new Float64Array( len );
	for ( i = 0; i < len; i++ ) {
		x[ i ] = uniform( -10.0, 10.0 );
		min[ i ] = uniform( -20.0, 0.0 );
		max[ i ] = uniform( min[ i ], min[ i ] + 40.0 );
	}

```

This would be the correct way to initialize random values for the variables.

Reply:
Thanks for this update. This really helps!!!